.gitattributes

```diff
@@ -1,2 +1,2 @@
-.png filter=lfs diff=lfs merge=lfs -text
 *.png filter=lfs diff=lfs merge=lfs -text
+*.ttf filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why is there a change here for PNGs?

Reply:
The purpose was to track all files, having extension as ".png"  (or to track "*.png") with git lfs. There was a typo, and I ended up tracking the exact files with name: ".png" instead. The change here corrects it.